client/stylesheets/_toCat.scss

```diff
@@ -5401,6 +5413,10 @@ On click the images will be displayed at normal size to complete the effect
 	margin-right: 12px;
 }
 
+ul button a {
+    font-family: 'Helvetica Neue', sans-serif !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces